sockeye/translate.py

```diff
@@ -18,25 +18,28 @@
 import sys
 import time
 from contextlib import ExitStack
+from typing import Optional
 
 import mxnet as mx
 
 import sockeye.arguments as arguments
 import sockeye.data_io
 import sockeye.inference
 import sockeye.output_handler
+from sockeye.inference import Translator
```

Comment:
duplicate import to "import sockeye.inference".
We tend to only import the module not individual names across the project, so I'd go with sockeye.inference.Translator in the code (and in the type annotation you added)